scenarios/reproducers/networking-definition-ceph.yml

```diff
@@ -0,0 +1,132 @@
+---
+cifmw_networking_definition:
+  networks:
+    ctlplane:
+      network: "192.168.122.0/24"
+      gateway: "192.168.122.1"
+      dns:
+        - "192.168.122.1"
+      mtu: 1500
+      tools:
+        multus:
+          ranges:
+            - start: 30
+              end: 70
+        netconfig:
+          ranges:
+            - start: 100
+              end: 120
+            - start: 150
+              end: 200
+        metallb:
+          ranges:
+            - start: 80
+              end: 90
+    internalapi:
+      network: "172.17.0.0/24"
+      vlan: 20
+      mtu: 1496
+      tools:
+        metallb:
+          ranges:
+            - start: 80
+              end: 90
+        netconfig:
+          ranges:
+            - start: 100
+              end: 250
+        multus:
+          ranges:
+            - start: 30
+              end: 70
+    storage:
+      network: "172.18.0.0/24"
+      vlan: 21
+      mtu: 1496
+      tools:
+        metallb:
+          ranges:
+            - start: 80
+              end: 90
+        netconfig:
+          ranges:
+            - start: 100
+              end: 250
+        multus:
+          ranges:
+            - start: 30
+              end: 70
+    tenant:
+      network: "172.19.0.0/24"
+      tools:
+        metallb:
+          ranges:
+            - start: 80
+              end: 90
+        netconfig:
+          ranges:
+            - start: 100
+              end: 250
+        multus:
+          ranges:
+            - start: 30
+              end: 70
+      vlan: 22
+      mtu: 1496
+    external:
+      network: "10.0.0.0/24"
+      tools:
+        netconfig:
+          ranges:
+            - start: 100
+              end: 250
+      vlan: 22
+      mtu: 1500
+    storagemgmt:
+      network: "172.20.0.0/24"
+      tools:
+        netconfig:
+          ranges:
+            - start: 100
+              end: 250
+      vlan: 23
+      mtu: 1500
+
+  group-templates:
+    ocps:
+      network-template:
+        range:
+          start: 10
+          length: 5
+      networks:
+        ctlplane: {}
+        internalapi: {}
+        tenant: {}
+        storage: {}
+    computes:
+      network-template:
+        range:
+          start: 100
+          length: 21
+      networks:
+        ctlplane: {}
+        internalapi: {}
+        tenant: {}
+        storage: {}
+        storagemgmt: {}
+    cephs:
```

Comment:
Instead of introducing `networking-definition-ceph.yml`, what would happen if you just added the `cephs` definition to the existing `networking-definition.yml` (since that is the only way they differ [1])? 

If the presence of the `cephs` structure doesn't break VA1, I think it would be better to let it have a single networking definition with an item that is not used so there is less reproduced content.

Also, if `cephs` is just for nodes which host ceph then, please you could remove `internalapi` and `tentant` network.

[1] 
```
$ diff -u ../ci-framework/scenarios/reproducers/networking-definition.yml networking-definition-ceph.yml
--- ../ci-framework/scenarios/reproducers/networking-definition.yml	2024-03-04 11:41:06.757530796 -0500
+++ networking-definition-ceph.yml	2024-03-08 07:55:24.749427209 -0500
@@ -114,6 +114,17 @@
         tenant: {}
         storage: {}
         storagemgmt: {}
+    cephs:
+      network-template:
+        range:
+          start: 150
+          length: 21
+      networks:
+        ctlplane: {}
+        internalapi: {}
+        tenant: {}
+        storage: {}
+        storagemgmt: {}
   instances:
     controller-0:
       networks:
```

Reply:
+1, that will work. If the Networking Definition is used in a deployment without one of the groups been a group in the Ansible inventory nothing will break.